### PR TITLE
Resolved signup success big screen layout issue

### PR DESCRIPTION
**Pull request by patelravi**

### CODE REVIEW

File: front/components/dialogs/shareIdeaByEmail.vue [hunk 1]
```diff
@@ -83,7 +90,21 @@ export default {
...
 
-      this.$emit('success')
+      // Call shre idea over email api.
+
+      this.closeDialog()
+    },
+    closeDialog() {
+      this.$emit('close')
+
+      setTimeout(() => {
```
Comment (aldarund) [inline front/components/dialogs/shareIdeaByEmail.vue]: why timeout? Why not requestAnimationFrame without timeout ?

File: front/pages/settings/index.vue [hunk 2]
```diff
@@ -9,54 +9,46 @@
       <div class="headerOfSetting">
         SETTINGS
       </div>
+
       <v-container class="settingsList">
-        <div class="settingsItem">
+        <router-link class="settingsItem" :to="{ name: 'profile' }">
```
Comment (aldarund) [inline front/pages/settings/index.vue]: use nuxt-link please

File: front/pages/settings/index.vue [hunk 3]
```diff
@@ -9,54 +9,46 @@
...
-        <div class="settingsItem">
-          <div class="settingsInfo">
-            <v-icon>fas fa-bell</v-icon>Notifications
-          </div>
-          <div class="metadata">All</div>
-        </div>
-        <div class="settingsItem">
+        </router-link>
+
+        <router-link class="settingsItem" :to="{ name: 'settings-email' }">
```
Comment (aldarund) [inline front/pages/settings/index.vue]: use nuxt-link please

File: front/components/dialogs/shareIdeaByEmail.vue (hunk 1 above)
Comment (patelravi) [reply front/components/dialogs/shareIdeaByEmail.vue]: I haven't used it before, can you provide one example?

File: front/pages/settings/index.vue (hunk 3 above)
Comment (patelravi) [reply front/pages/settings/index.vue]: Made the changes.

File: front/pages/settings/index.vue (hunk 2 above)
Comment (patelravi) [reply front/pages/settings/index.vue]: Made the changes.

**Pull request merged**